cfn-resources/project-invitation/cmd/resource/resource.go

```diff
@@ -0,0 +1,247 @@
+package resource
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/handler"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/constants"
+	log "github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
+	progressevents "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+var CreateRequiredFields = []string{constants.ProjectID, constants.PubKey, constants.PvtKey, constants.Username}
+var ReadRequiredFields = []string{constants.ProjectID, constants.ID, constants.PubKey, constants.PvtKey}
+var UpdateRequiredFields = []string{constants.ProjectID, constants.ID, constants.PubKey, constants.PvtKey}
+var DeleteRequiredFields = []string{constants.ProjectID, constants.ID, constants.PubKey, constants.PvtKey}
+var ListRequiredFields = []string{constants.PubKey, constants.PubKey}
+
+func validateModel(fields []string, model *Model) *handler.ProgressEvent {
+	return validator.ValidateModel(fields, model)
+}
+
+func setup() {
+	util.SetupLogger("mongodb-atlas-project-invitation")
+}
+
+func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler.ProgressEvent, error) {
+	setup()
+
+	_, _ = log.Debugf("Create() currentModel:%+v", currentModel)
```

Comment:
```suggestion
	 log.Debugf("Create() currentModel:%+v", currentModel)
```

Reply:
the same in all the others

Reply:
on implementing this suggested change, our lint checks fails.

Reply:
okok we need to see in the future if we can delete the return of those logs, if we are not going to use them